data/data/powervs/cluster/power_network/variables.tf

```diff
@@ -29,3 +29,8 @@ variable "cloud_conn_name" {
   description = "The name of a pre-created Power VS Cloud connection."
   default     = ""
 }
+
+variable "machine_cidr" {
```

Comment:
Wouldn't this cause an error since it's not being set and has no default? I can't test it so relying on you guys

Reply:
the default is embedded in the functionality itself so this is safe. additionally, i'd rather not set a default here in case powervs changes it and they're out of sync

Reply:
sorry, this isn't entirely accurate and i realized it when i was rebasing. there is a default (pointed out in my other comment). that will always be set in the install-config file and passed to this module.